test/e2e/storage/framework/volume_resource.go

```diff
@@ -204,7 +204,7 @@ func (r *VolumeResource) CleanupResource() error {
 				}
 
 				if pv != nil {
-					err = e2epv.WaitForPersistentVolumeDeleted(f.ClientSet, pv.Name, 5*time.Second, 5*time.Minute)
+					err = e2epv.WaitForPersistentVolumeDeleted(f.ClientSet, pv.Name, 5*time.Second, f.Timeouts.PVDelete)
```

Comment:
The default value for `f.Timeouts.PVDelete` is 3 minutes. I can optionally make it 5.

Reply:
Looking through a [recent test run](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/cloud-provider-gcp/314/cloud-provider-gcp-e2e-full/1508801888009588736/build-log.txt), PVCs seem to be deleted by the first 5s poll (search through the build log for "Deleting pvc"). So I think it should be fine to drop the timeout to 3m.

/lgtm